test/kwtool_web/controllers/auth_controller_test.exs

```diff
@@ -0,0 +1,10 @@
+defmodule KwtoolWeb.AuthControllerTest do
+  use KwtoolWeb.ConnCase
```

Comment:
Let's add `async: true` here and for other tests also

Reply:
hi @hanam1ni !
Could you please share why we need to add async here? :)

Reply:
This option allow the tests to run concurrently

![image](https://user-images.githubusercontent.com/14077479/112929745-37cdc280-9143-11eb-8fb7-c11e7d61ed44.png)